File: docs/src/testers/sessions/cdp/playwright_timeout.py
# @sniptest filename=playwright_timeout.py
# @sniptest show=1-7
from notte_sdk import NotteClient

client = NotteClient()

with client.Session(idle_timeout_minutes=20, max_duration_minutes=20) as session:
    # Long Playwright automation
    pass
    status = session.status()

File: docs/src/testers/sessions/cdp/playwright_timeout.ts
// @sniptest filename=playwright_timeout.ts
// @sniptest show=1-8
import { NotteClient } from 'notte-sdk';

const client = new NotteClient();

const status = await client.Session({ idle_timeout_minutes: 20, max_duration_minutes: 20 }).use(async session => {
  // Long Playwright automation
  return session.status();
});

export { status };
